src/Uno.Playground.Shared/Styles/Controls/AppBarButton.xaml

```diff
@@ -227,7 +227,7 @@
 		</Setter>
 	</win:Style>
 
-	<Style TargetType="AppBarButton"
-		   BasedOn="{StaticResource NativeDefaultAppBarButton}" />
+	<!--<Style TargetType="AppBarButton"
```

Comment:
Can you explain why it's commented here please ;) ? 
Don't need it anymore ? Related to an issue ?

Reply:
We now support the WinUI AppBarButton, so native style is not needed.